blog/2015/10/weekly-update-2015-10-26.html

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+root: ../../..
+author: Anelda van der Walt
+title: "October 19 - 26, 2015: 500 Workshops & 16,000 Participants, Retreat, Debriefing Sessions, 
+Digital Data Storage, and A Science Competition."
+date: 2015-10-26
+time: "14:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p><strong>Highlights</strong></p>
+<ul>
+  <li>
+    The Software Carpentry team has now run more than 
+    <a href="{{page.root}}/blog/2015/10/recent-stats.html">500 workshops for 16,000 participants</a>!
+  </li>
+  <li>
+    Remember to join our 
+    <a href="{{page.root}}/blog/2015/09/instructor-helper-retreat.html">global instructor/helper retreat</a>
+    on 14 November. 
+    If you care about training, join a site near you or dial in via Google Hangouts.
+  </li>
+</ul>
+<p><strong>Dates</strong></p>
+<ul>
+  <li>
+    Please take note of the planned dates for upcoming 
+    <a href="{{page.root}}/blog/2015/10/next-debriefings.html">debriefing sessions for instructors</a>.
+  </li>
+</ul>
+<p><strong>Publications</strong></p>
+<ul>
+  <li>
+    <a href="{{page.root}}/blog/2015/10/ten-simple-rules-for-digital-data-storage.html">
+    Ten Simple Rules for Digital Data Storage</a>: 
```

Comment:
No line break after the opening tag (inserts an unwanted space in the generated page).